test/kernels/test_rbf_kernel.py

```diff
@@ -11,6 +11,9 @@ class TestRBFKernel(unittest.TestCase, BaseKernelTestCase):
     def create_kernel_no_ard(self, **kwargs):
         return RBFKernel(**kwargs)
 
+    def create_kernel_ard(self, num_dims, **kwargs):
+        return RBFKernel(ard_num_dims=num_dims, **kwargs)
+
     def test_ard(self):
```

Comment:
Do we need this test if we have BaseKernelTestCase?

Reply:
I think it's nice to have because it doesn't explicitly assume that at least the most basic kernel call is correct. All of the test cases we can write for `BaseKernelTestCase` are basically limited to testing complex kernel calls against simpler ones (e.g., a batch mode call vs filling in the batch kernel matrix in for loops).

Reply:
Could we rename it to something more generic then?